documentation/modules/exploit/multi/browser/chrome_cve_2021_21220_v8_insufficient_validation.md

```diff
@@ -0,0 +1,168 @@
+## Vulnerable Application
+
+This module exploits CVE-2021-21220 an out of bounds access issue in Google Chrome versions before
+89.0.4389.128/90.0.4430.72 (64 bit) that was reported as part of Pwn2Own 2021 by Bruno Keith and
+Niklas Baumstark of Dataflow Security.
+
+The exploit makes takes advantage of a lack of proper input validation in the V8 engine on x86_x64 builds
+of Chrome when handling XOR operations within JIT compiled code. Successful exploitation leads to RCE execution
+within the context of the V8 renderer.
+
+Note that as the V8 renderer is sandboxed within typical Chrome setups, the browser must be run without a sandbox by using
+the --no-sandbox option for the payload to work correctly.
+
+The module is compatible with any 64bit Google Chrome (versions before 87.0.4280.88) on multiple platforms.
+However, the code that writes the shellcode into the rwx region (wasm_rwx_addr) may need to be modified.
+
+**Vulnerable Application Installation Steps**
+
+You can download a vulnerable Chrome version from this location:
+[https://chromium.cypress.io/win64/beta/90.0.4430.70](https://chromium.cypress.io/win64/beta/90.0.4430.70)
+
+## Verification Steps
+
+1. Do: `use exploit/multi/browser/chrome_simplifiedlowering_overflow`
```

Comment:
s/chrome_simplified_lowering_overflow/chrome_cve_2021_21220_v8_insufficient_validation

Reply:
Ah thanks good spotting will get this resolved 👍